src/plots/cartesian/axes.js

```diff
@@ -1207,7 +1207,7 @@ axes.getSubplots = function(gd, ax) {
     var i, j, sp;
 
     // look for subplots in the data
-    var data = gd.data || [];
+    var data = gd._fullData || gd.data || [];
```

Comment:
I didn't want to break

![image](https://cloud.githubusercontent.com/assets/6675409/19651254/1872472c-99da-11e6-9799-90ae040845d9.png)